### DISCUSSION TITLE
Create custom layer reference

### DISCUSSION THREAD

**Discussion by laurapigeon** [manna-harbour/miryoku#282] [Q&A] [answered]
At [the reference manual for default miryoku](https://github.com/manna-harbour/miryoku/tree/master/docs/reference), and [this custom layout example](https://github.com/manna-harbour/miryoku/discussions/134), there are pngs summarising the keys. How do I make such an image for my custom layout, for my own reference and for when I post it here as an example? Can it be generated from my `custom_config.h` or created from scratch at a certain URL? Thanks!

**Comment by manna-harbour** [comment, marked as answer]
You can edit the current diagrams. See https://github.com/manna-harbour/miryoku/discussions/71#discussioncomment-2108278.

The diagrams will be auto generated in future. See https://github.com/manna-harbour/miryoku/issues/72. That currently supports individual layer diagrams only, and doesn't yet support the combined layers diagram. Miryoku SVG hasn't been released yet, but diagrams will be generated by private request.

`custom_config.h` is per implementation, so you'd need to also make one for Miryoku SVG with your changes. See https://github.com/manna-harbour/miryoku_babel/tree/master/tangled/svg for the original layer data. 

Alternatively use https://github.com/manna-harbour/miryoku_babel directly to make your changes, then you won't need to maintain `custom_config.h` for multiple implementations.
